### DISCUSSION TITLE
How to get data from the ladar?

### DISCUSSION THREAD

**Discussion by ghost** [iliasam/OpenSimpleLidar#12] [Q&A] [answered]
Hi! I'm a really fresh man. How could I get the data from the ladar on PC?
 (ie in the Open Simple Lidar Calibration step. I see in the video I need a usb-uart module, but how can I connect it to the radar? The data from motor pcb to the moldule's RXI?)

**Comment by iliasam** [comment, marked as answer]
See the schematic:
https://github.com/iliasam/OpenSimpleLidar/blob/master/PCB/CommonPCB_v1.2M/Schematic_motor_v1.2.png
You need to connect your USB-UART (3.3V) adapter to the pads of "Ext Lines". 
Adapter RX is need to be connected to DATA of "Ext Lines".
Adapter GND is need to be connected to EXT_GND of "Ext Lines".
So finally Adapter RX will be connected to the TX line of the MCU - PA2.

P.S. If you are "a really fresh man", are you sure that you can solder TSL1401CL to the PCB?

**Reply by ghost** [reply]
Thank you! I have little experience.I've been exposed to soldering, albeit with little experienc.Hopefully, it will work
